apps/web/src/components/Basenames/RegistrationForm/index.tsx

```diff
@@ -395,7 +400,6 @@ export default function RegistrationForm() {
             baseSingleYearEthCost={singleYearBasePrice}
             isOpen={premiumExplainerModalOpen}
             toggleModal={togglePremiumExplainerModal}
-            nameLength={selectedName?.length}
```

Comment:
why are we removing this?

Reply:
The prop doesn't exist on the component. It's a typeError.

Reply:
makes sense, I thought we were using that but I guess it's now obsolete.